torch/_higher_order_ops/while_loop.py

```diff
@@ -0,0 +1,126 @@
+import torch.utils._pytree as pytree
+
```

Comment:
btw, in torch/_higher_order_ops/__init__.py, we should import all the higher order ops so that they are already usable via torch.ops.higher_order, otherwise the UX for accessing these is a bit weird. `import torch` should import `torch._higher_order_ops`

Reply:
Ok, can do it in a separate PR!